.github/workflows/publish-maven-snapshots.yml

```diff
@@ -5,8 +5,8 @@ on:
   push:
       branches:
         - main
-        - '1.3'
-        - 2.x
+        - '[0-9]+.[0-9]+'
+        - '[0-9]+.x'
```

Comment:
We also want to be publishing via `2.minor` branch example, 2.7, 2.8, etc.

Reply:
Line 8 will cover that